packages/algolia/schema/crn/algolia-schema.json

```diff
@@ -44,7 +44,8 @@
     "interestGroup.id",
     "workingGroup.id",
     "workingGroups.id",
-    "publishingEntity"
+    "publishingEntity",
+    "complete"
```

Comment:
Just wondering why we add the complete field to the attributesToIndex.

Reply:
nice! you're right. I thought it was necessary to add it to `attributesToIndex` as well so the filter could work, but it is not necessary